config/production.json

```diff
@@ -45,6 +45,7 @@
 		"manage/security": true,
 		"manage/site-settings/analytics": true,
 		"manage/site-settings/categories": true,
+		"manage/site-settings/disconnect-flow": false,
```

Comment:
MINOR: I believe convention is to omit these until they're activated, but I'm not sure.

If we _do_ add the `false` configs, we should probably add them in the other configs to be consistent.